.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,6 +30,15 @@
 
 :beginner: If you have a lot to say, be more detailed in this section.
 
+# Testing
+<!--
+    Talk about the testing you have (and have not) done. It's useful
+    for others to know what you've already checked so that they can
+    avoid repeating the same things and consider what you might have missed.
+-->
+
+:beginner: If you tested on specific pages, mention them here (such as
+“Tested on `/release/mbid` using a sample/full database”).
```

Comment:
* Slightly reformat to make it easier to copy/paste and edit.
* Mention _what_ is being tested on the pages.

```suggestion
:beginner: If you tested anything on specific pages, mention them here, such as:
Tested something on `/release/mbid` using a sample/full database
```